app/scripts/background.js

```diff
@@ -273,6 +273,19 @@ function saveTimestamp() {
  * @returns {Promise} Setup complete.
  */
 async function initialize() {
+  try {
+    // https://bugs.chromium.org/p/chromium/issues/detail?id=1457040
+    // Temporary workaround for chromium bug that breaks the content
+    // script <=> background connection for prerendered pages.
+    // eslint-disable-next-line no-undef
+    chrome?.privacy.network.networkPredictionEnabled.set({
+      value: true,
```

Comment:
What does calling this do? Per https://developer.chrome.com/docs/extensions/reference/api/privacy, this value seems to already be `true` by default.

Reply:
good catch, i wasn't thinking and changed this to true after my initial testing. Will change back to false

Reply:
[1ec9284](https://github.com/MetaMask/metamask-extension/pull/24201/commits/1ec9284ab902d69bd6ff03a46638f78d084aae22)